src/borg/helpers.py

```diff
@@ -2277,3 +2277,10 @@ def dash_open(path, mode):
 
 def is_terminal(fd=sys.stdout):
     return hasattr(fd, 'isatty') and fd.isatty() and (sys.platform != 'win32' or 'ANSICON' in os.environ)
+
+
+def umount(mountpoint):
+    try:
+        return subprocess.call(['fusermount', '-u', mountpoint])
```

Comment:
is there `fusermount` on anything else than linux?

would work, but maybe a bit unpretty. if we ever have windows support with fuse, we also would likely have to move it back to platform package.